testgen/icu75/message_fmt2/formatter/integer.json

```diff
@@ -0,0 +1,55 @@
+{
+  "scenario": "Integer formatter",
+  "defaultTestProperties": {
+    "testSubtype": "formatter",
+    "locale": "en-US"
+  },
+  "tests": [
+    {
+      "pattern": "hello {4.2 :integer}",
+      "verify": "hello 4"
+    },
+    {
+      "pattern": "hello {-4.20 :integer}",
+      "verify": "hello -4"
+    },
+    {
+      "pattern": "hello {0.42e+1 :integer}",
+      "verify": "hello 4"
+    },
+    {
+      "pattern": ".match {$foo :integer} one {{one}} * {{other}}",
+      "args": [
+        {
+          "argType": "number",
+          "name": "foo",
+          "value": 1.2
+        }
+      ],
+      "verify": "one"
+    },
+    {
+      "pattern": ".match {$foo :integer} 1.2 {{=1.2}} one {{one}} * {{other}}",
+      "args": [
+        {
+          "argType": "number",
+          "name": "foo",
+          "value": 1.2
+        }
+      ],
+      "verify": "=1.2"
+    },
+    {
+      "pattern": ".match {$foo :integer} 1.2 {{=1.2}} |1,2| {{=1,2}} * {{other}}",
+      "locale": "fr",
+      "args": [
+        {
+          "argType": "number",
+          "name": "foo",
+          "value": 1.2
+        }
+      ],
+      "verify": "=1.2"
```

Comment:
Ditto, but double problem: formatting should return `1`. The matching ought not to happen on the localized formatted string, but even if it did, the resulting string would be `=1,2`, right? Also, does our syntax support `,` in a number literal? It [doesn't seem so](https://github.com/unicode-org/message-format-wg/blob/main/spec/syntax.md#literals).